src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Markdown.cs

```diff
@@ -51,7 +78,7 @@ public static DiagnosticData AddMarkdownView(this Response response, string mark
         /// }
         /// </code>
         /// </example>
-        public static DiagnosticData AddMarkdownView(this Response response, string markdown, bool enableEmailButtons, string title = null)
+        public static DiagnosticData AddMarkdownView(this Response response, string markdown, bool enableEmailButtons, string title = null, bool isContainerNeeded = true)
```

Comment:
This is a breaking change as you are modifying the existing definition.

Reply:
Correct me if I'm wrong but adding the optional parameter does not break detectors calling this method.

Reply:
it will break as method signature has changed. Pre-compiled versions will break.